package-lock.json

```diff
@@ -1,15376 +0,0 @@
-{
```

Comment:
Is there a new `package-lock.json` file we could commit? I'd rather not go without one due to CI and all that.